File: program/convert-string-to-pascalcase/ConvertStringToPascalcase.cs
using System;
using System.Globalization;

public class ConvertStringToPascalCase{
    public static void Main(){
        Console.WriteLine("Enter a string to convert it to PascalCase: ");
        string demoInput = Console.ReadLine();
        Console.WriteLine(converStringToPascalCase(demoInput).Replace(" ",""));
    }
    
    static string converStringToPascalCase(string s){
        TextInfo myTI = new CultureInfo("en-US",false).TextInfo;
        return myTI.ToTitleCase(s);
    }
}

File: program/program/convert-string-to-pascalcase/ConvertStringToPascalcase.fs
open System
open System.Globalization


let convertStringToPascalCase (s: string) =
    let myTI = CultureInfo("en-US", false).TextInfo
    myTI.ToTitleCase(s).Split(' ') |> Array.map (fun x -> x.Trim())

[<EntryPoint>]
let main argv =
    printfn "Enter a string to convert it to PascalCase: "
    let demoInput = Console.ReadLine()
    let pascalCaseString = convertStringToPascalCase demoInput |> String.concat ""
    printfn "%s" pascalCaseString
    0
